torch/csrc/jit/passes/freeze_module.cpp

```diff
@@ -11,6 +12,20 @@ namespace torch {
 namespace jit {
 
 namespace {
+ModulePtr getModulePtrForGetAttrNode(
```

Comment:
Can't we use findConstantAttr?

Reply:
We could. It would require some refactoring of that function, because it does more than just follow getAttr chain. If `getModulePtrForGetAttrNode ` was too big I would probably do that. If you prefer that I can think of refactoring that.

Reply:
It am okay to keep them separate for now. There is some duplication of code but not too much....